internal/grpc/server.go

```diff
@@ -19,14 +19,20 @@ import (
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 
+	"github.com/grpc-ecosystem/go-grpc-prometheus"
+
 	v2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
 	discovery "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v2"
 	loadstats "github.com/envoyproxy/go-control-plane/envoy/service/load_stats/v2"
 	"github.com/sirupsen/logrus"
 )
 
+var ServerMetrics = grpc_prometheus.NewServerMetrics()
```

Comment:
Please don't use a package global here. Can you please move this to a field on the `grpcServer` server type. Thank you.

Reply:
Done! I had to pass down the prom `Registry` to the constructor to keep `grpcServer` private. Let me know what you think.